drivers/include/ksz8851snl.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
```

Comment:
You are probably not Kaspar Schleiser, right? Although Kaspar might be happy that you donate him your copyright :-)